pkg/k8s/pods/pod.go

```diff
@@ -178,6 +178,54 @@ func Restart(dev *model.Dev, c *kubernetes.Clientset) error {
 	return waitUntilRunning(dev.Namespace, fmt.Sprintf("%s=%s", okLabels.DetachedDevLabel, dev.Name), c)
 }
 
+// RunCleanerPod runs a pod to clean the dev environmentt volume
+func RunCleanerPod(dev *model.Dev, c *kubernetes.Clientset) error {
```

Comment:
It feels like a `job` would be more suitable for this than the pod no?

Reply:
Not really. The job would create a pod, and we would need to monitor the pod anyway, and clean the pod and the job. jobs are suitable for cronjobs or tasks where you want to run up to 1000 tasks with less than X errors.